CMakeLists.txt

```diff
@@ -123,8 +124,11 @@ set(SWIFT_INSTALL_COMPONENTS "${_SWIFT_KNOWN_INSTALL_COMPONENTS}" CACHE STRING
 # * toolchain-dev-tools -- install development tools useful in a shared toolchain
 # * dev -- headers and libraries required to use Swift compiler as a library.
 
-set(SWIFT_SDKS "" CACHE STRING
-    "If non-empty, limits building target binaries only to specified SDKs (despite other SDKs being available)")
+set(SWIFT_CROSS_COMPILE_STDLIB_TARGETS "" CACHE STRING
+    "A list of deployment-targets to cross-compile the standard-library for (by default, the standard library is always compiled for the host)")
+
+set(SWIFT_SKIP_HOST_STDLIB "" CACHE STRING
+  "If True, will not build the standard library for the host")
```

Comment:
Maybe I didn't describe it well, but what I meant to say is that (currently, before merging this PR) we always build for the host even if you don't include it in SWIFT_SDKS. So it doesn't actually work today (SWIFT_PRIMARY_VARIANT will never be Android).

While doing this patch, I removed the part that always builds for the host and found that there were other places which do assume the host is always being built. Maybe there are some other combination of settings which make it work (not building those failing parts)? I don't know; so I added the flag. It's not necessary though.